src/plugin.js

```diff
@@ -138,21 +128,41 @@ var annotationPlugin = {
 				});
 			});
 		});
+
+		// Detect and intercept events that happen on an annotation element
+		var config = chartInstance.options.annotation || {};
+		if (config.events) {
```

Comment:
[info] just a quick bit of info here. This will become a lot easier once some of the planned Chart.js v2.5 work is done. The goal is to have plugins notified about events so that you don't have to add your own listeners.